btc_parser/sources/block.move

```diff
@@ -10,7 +10,7 @@ use btc_parser::tx::{Self, Transaction};
 use std::unit_test::assert_eq;
 
 /// A block is a collection of all transactions in the BTC block
-public struct Block has copy, drop {
+public struct Block has copy, drop, store {
```

Comment:
let's try to remove copy - we should use reference, rather than copy it (especially, that this contains transactions vector)

Reply:
No, reference is developer choice, we can use &T without problem.
Copy is automatic without &.

Reply:
Yes, for better dev experience.